spec/models/workshop_spec.rb

```diff
@@ -16,13 +16,29 @@
       end
 
       it 'validate if chapter_id present' do
-        workshop.chapter_id = 1
+        workshop.chapter = Fabricate(:chapter)
```

Comment:
I believe the reason we use the chapter id is to simulate conrtoller behavior which sets the chapter_id on the workshop model before ti attempts to save.